templates/keras_nlp/index.md

```diff
@@ -2,20 +2,25 @@
 
 <a class="github-button" href="https://github.com/keras-team/keras-nlp" data-size="large" data-show-count="true" aria-label="Star keras-team/keras-nlp on GitHub">Star</a>
 
-KerasNLP is a simple and powerful API for building Natural Language Processing
-(NLP) models within the Keras ecosystem.
-
-KerasNLP provides modular building blocks following standard Keras interfaces
-(layers, metrics) that allow you to quickly and flexibly iterate on your task.
-Engineers working in applied NLP can leverage the library to assemble training
-and inference pipelines that are both state-of-the-art and production-grade.
-
-KerasNLP can be understood as a horizontal extension of the Keras API:
-components are first-party Keras objects that are too specialized to be
-added to core Keras, but that receive the same level of polish as the rest of
-the Keras API.
-
-KerasNLP is also new and growing! If you are interested in contributing, please
+KerasNLP is a natural language processing library that supports users through
+their entire development cycle. Our workflows are built from modular components 
+that have state-of-the-art preset weights and architectures when used 
+out-of-the-box and are easily customizable when more control is needed. We 
+emphasize in-graph computation for all workflows so that developers can expect 
+easy productionization using the Tensorflow ecosystem.
+
+This library is an extension of the core Keras API; all high-level modules are 
+[`Layers`](https://keras.io/api/layers/) or 
```

Comment:
I think you can just do `/api/layers/` and ditto for other links on this page. Then they will stay correct for local previews as well.